proto/hyperlane/warp/v1/tx.proto

```diff
@@ -77,6 +80,22 @@ message MsgCreateSyntheticToken {
   ];
 }
 
+// MsgCreateNativeSyntheticToken ...
+message MsgCreateNativeSyntheticToken {
+  option (cosmos.msg.v1.signer) = "owner";
+  option (amino.name) = "hyperlane/warp/v1/MsgCreateNativeSyntheticToken";
+
+  string owner = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+
+  string origin_mailbox = 2 [
+    (gogoproto.customtype) =
+        "github.com/bcp-innovations/hyperlane-cosmos/util.HexAddress",
+    (gogoproto.nullable) = false
+  ];
+
+  string origin_denom = 3;
```

Comment:
would "local" be a better descriptor than "origin" here? theres a lot of overloading of nomenclature here but I think we typically use origin in the context of interchain messages

Reply:
I've made the change in https://github.com/bcp-innovations/hyperlane-cosmos/pull/135/commits/ac19a81d35cca5faf11234211cefa1631c91f19b, but the reason I called it "origin" is because the `HypToken` interface calls it that!

Reply:
I see, apologies
I was confused whether this was intended for some crosschain factory
I think we should be consistent